src/linear_algebra/affine_space/independent.lean

```diff
@@ -734,6 +734,37 @@ by { ext, simp [face_points] }
   {m : ℕ} (h : fs.card = m + 1) : set.range (s.face h).points = s.points '' ↑fs :=
 by rw [face_points', set.range_comp, finset.range_order_emb_of_fin]
 
+/-- Remap a simplex along an `equiv` of index types. -/
+@[simps]
+def reindex {m n : ℕ} (s : simplex k P m) (e : fin (m + 1) ≃ fin (n + 1)) : simplex k P n :=
+⟨s.points ∘ e.symm, (affine_independent_equiv e.symm).2 s.independent⟩
```

Comment:
This works more generally for an `embedding` or a `embedding_class`, right?

Reply:
Well, we have `face` to extract a simplex with a given subset of points, but yes, it would be possible to define a variant using an embedding and derive various things from that. It would be good to have proper documentation on the leanprover-community website of conventions for reindexing and similar operations (and ensure existing APIs follow them). For example, when should the operation be `reindex` (`basis`) versus `comp_equiv` (`affine_basis`) versus `dom_dom_congr` (`multilinear_map`)? Which way round should an `equiv` of index types be specified, when an `equiv` is used?